Gopkg.toml

```diff
@@ -16,7 +16,7 @@ required = [
 [[constraint]]
   # Also defined in WORKSPACE
   revision = "f8815663de1e64d57cdd4ee9e2b2fa96977a030e"
-  name = "github.com/go-resty/resty"
+  name = "gopkg.in/resty.v1"
```

Comment:
Does this still work? I think since our generated files use the other import we still need to include both. I'm not sure this is a complete fix. The correct way to fix this would be to correct the generated files as well. Sorry I know that means it will be harder to get done.